pkg/elemental/edit/elemental.cattle.io.managedosversionchannel.vue

```diff
@@ -0,0 +1,64 @@
+<script>
+import Loading from '@shell/components/Loading.vue';
+import CruResource from '@shell/components/CruResource.vue';
+import CreateEditView from '@shell/mixins/create-edit-view';
+import { LabeledInput } from '@components/Form/LabeledInput';
+import NameNsDescription from '@shell/components/form/NameNsDescription';
+
+export default {
+  name:       'ManagedOsVersionChannelEditView',
+  components: {
+    Loading, LabeledInput, CruResource, NameNsDescription
+  },
+  mixins:     [CreateEditView],
+  props:      {
+    value: {
+      type:     Object,
+      required: true
+    },
+    mode: {
+      type:     String,
+      required: true
+    },
+  },
+  async fetch() {},
+  data() {
+    return {};
+  },
+  computed: {},
+  methods:  {},
+};
+</script>
+
+<template>
+  <Loading v-if="!value" />
```

Comment:
If it's possible here you could use `v-if="$fetchState.pending"` to have a more accurate representation of the resource being loaded.